src/main/java/uk/gov/hmcts/reform/demo/DemoApplication.java

```diff
@@ -0,0 +1,13 @@
+package uk.gov.hmcts.reform.demo;
+
+import org.springframework.boot.SpringApplication;
+import org.springframework.boot.autoconfigure.SpringBootApplication;
+
+// TODO: change the name of this class to one matching your project, e.g. DraftStoreApplication
```

Comment:
isn't just `Application.java` fine for all projects?

Reply:
That was my first plan, but then I saw DraftStoreApplication in draft-store (don't have access to internal repositories).